spec/controllers/evidence_controller_spec.rb

```diff
@@ -167,6 +168,36 @@
     end
   end
 
+  describe 'GET #summary' do
+    before { get :summary, id: evidence }
+
+    it 'returns the correct status code' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template :summary
+    end
+  end
+
+  describe 'POST #summary_save' do
+    let(:user)     { create :user, office: create(:office) }
```

Comment:
@dotemacs In the evidence_controller_spec, I had to create the models
rather than stubbing, as there was no view model to mock and the 
current_user would need to be mocked and stubbed for one test.

Happy to discuss if you would rather these were stubbed, I wasn't sure on 
the time it would take/vs the time it would save!